src/pip/_internal/cli/base_command.py

```diff
@@ -213,6 +213,8 @@ def _main(self, args):
                 "This will become an error in pip 21.0."
             )
 
+        self.handle_pip_version_check(options)            
+        
```

Comment:
Remove whitespace to fix linting:

```suggestion
        self.handle_pip_version_check(options)

```